snuba/environment.py

```diff
@@ -38,7 +34,7 @@ def setup_sentry() -> None:
             RedisIntegration(),
         ],
         release=os.getenv("SNUBA_RELEASE"),
-        traces_sampler=traces_sampler,
+        traces_sample_rate=0,
```

Comment:
Okay so having the `traces_sample_rate=0` is the same as the `traces_sampler` return `False` correct?  

Do you know if `sampling_context["parent_sampled"]` was even doing anything to begin with? 

Do we know what

Reply:
Your understanding is correct, according to Markus the default SDK/ingest behavior was to send if `sampling_context["parent_sampled"]` was truthy and default to not sending a trace otherwise.

Reply:
yup exactly